test/flop_test.exs

```diff
@@ -772,6 +778,45 @@ defmodule FlopTest do
     end
   end
 
+  describe "cursor paging" do
+    test "cursor paging" do
+      insert_list(6, :pet)
+
+      {:ok, {r1, %Meta{end_cursor: end_cursor}}} =
```

Comment:
I think the tests should also make assertions on the exact items and order that you expect.